openapi.yaml

```diff
@@ -1421,6 +1474,92 @@ components:
         "protocol_parameters": { "k": 5, "m": 100, "phi_f": 0.65 }
         }
 
+    CardanoTransactionListMessage:
+      description: CardanoTransactionListMessage represents a list of Cardano transactions set
+      type: array
+      items:
+        type: object
+        additionalProperties: false
+        required:
+          - hash
+          - certificate_hash
+          - beacon
+          - created_at
+        properties:
+          hash:
+            description: Hash of the Cardano transactions set
+            type: string
+            format: bytes
+          certificate_hash:
+            description: Hash of the associated certificate
+            type: string
+            format: bytes
+          merkle_root:
+            description: Merkle root of the Cardano transactions set
+            type: string
+            format: bytes
+          beacon:
+            $ref: "#/components/schemas/Beacon"
+          created_at:
+            description: Date and time at which the Cardano transactions set was created
+            type: string
+            format: date-time,
+        example:
```

Comment:
In the example, it's supposed to be an array, isn't it?

Reply:
Well, this is the example for the array item in that case. 
That's why this is not an array :slightly_smiling_face:

Reply:
Oops, thanks 🙏

Reply:
So the name should not be `CardanoTransactionListItemMessage` instead of `CardanoTransactionListMessage`?

Reply:
And the type `object` instead of `array`?

Reply:
No the structure is an array of objects so it's normal that the name is `List` and not `ListItem`.
The object that represents the list item is not named here, this is probably what bothers you.

BTW, I simply followed the nomenclature already in place for the other signed entity types (e.g. see `MithrilStakeDistributionListMessage` definition above in the file)

Reply:
I got confused because`CertificateListMessage` and `SnapshotListMessage` have a different description than `MithrilStakeDistributionListMessage` and `CardanoTransactionListMessage`.